WordPress/src/main/java/org/wordpress/android/ui/CollapseFullScreenDialogFragment.java

```diff
@@ -56,6 +56,7 @@ public class CollapseFullScreenDialogFragment extends DialogFragment {
     private static final String ARG_HIDE_ACTIVITY_BAR = "ARG_HIDE_ACTIVITY_BAR";
     private static final String ARG_TITLE = "ARG_TITLE";
     private static final int ID_ACTION = 1;
+    private CollapseFullScreenDialogFragmentBinding mBinding;
```

Comment:
Reviewer Note (@ParaskP7): Should we always have refference of binding locally?.

also do you think I should making binding null on dismiss?

```
    @Override
    public void dismiss() {
        if (mHideActivityBar) {
            showActivityBar();
        }

        try {
            getParentFragmentManager().popBackStackImmediate();
        } catch (IllegalStateException e) {
            e.printStackTrace();
        }
    }
```

Reply:
👋 @neeldoshii !

> Should we always have refference of binding locally?

Since the `mBinding` is being used in both, the `onCreate(...)` and `onCreateView(...)` you can't avoid but using it as you did, via a shared field. Thus, the answer to your question is that "it depends".

> also do you think I should making binding null on dismiss?

I actually suggest doing that within the `onDestroyView()`, just like it is recommended by the docs, which is to use the `onDestroy()` lifecycle method(s) and remove any reference to the `mBinding` fields there.